docs/application-connector/docs/010-details-ac-deployment.md

```diff
@@ -3,59 +3,131 @@ title: Deploy a new Remote Environment
+
+The following operations are available:
+
+- Create a new Remote Environment
+- Delete the Remote Environment
+- Update Remote Environment configuration
+
+All operation can be done from within the `Console UI` or using `Kubectl` - command line tool.
+
+
+All Remote Environments are installed in the `kyma-integration` Namespace.
 
 >**NOTE:** A Remote Environment represents a single connected external solution.
 
-## Install a Remote Environment on a local Kyma deployment
 
-To install a new Remote Environment on Minikube, provide the NodePort as shown in this example:
+## Install a Remote Environment
 
-```
-helm install --name {remote-environment-name} --set deployment.args.sourceType=commerce --set global.isLocalEnv=true --set service.externalapi.nodePort=32001 --namespace kyma-integration ./resources/remote-environments
-```
+The Remote Environment can be removed from Kyma using either Console UI or Kubectl.
 
-You can override the following parameters:
+### Using the UI:
 
-- **sourceEnvironment** is the Event source Environment name.
-- **sourceType** is the Event source type.
-- **sourceNamespace** is the organization that publishes the Event.
+- Go to the Kyma console UI.
+- Select Administration.
+- Select the Remote Environments from the Integration section.
+- Click Create Remote Environment.
 
-Follow the **Set up a Remote Environment on Minikube** getting started guide to learn more about installing and setting up a Remote Environment on
-a local Kyma installation.
+![Add RE](./assets/create-re.png)
 
-## Install a Remote Environment on a cluster Kyma deployment
+- Provide the following details:
+    - Name
+    - Description
+    - Labels of your choice which are key-value pairs
+   
+![Update RE](./assets/edit-re.png)
 
-To add a new Remote Environment to the cluster, run this command:
+ - Click Create and the new Remote Environment will be created and you can check status in Remote Environment view.
+ 
 
-```
-helm install --name {remote-environment-name} --set deployment.args.sourceType=commerce --set global.isLocalEnv=false --set global.domainName={domain-name} --namespace kyma-integration ./resources/remote-environments
+### Using Kubectl
+
+New Remote Environment can be created using the Kubectl apply command with the following yaml file:
+
+re-production-1.yaml:
+
+``` yaml
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: RemoteEnvironment
+metadata:
+  name: production-1
+spec:
+  description: This Remote Environment for connecting production system 1.
+  labels:
+    region: us
+    kind: production
 ```
 
-The **global.domainName** is mandatory. Example values can look like:
+Then you can execute the following command:
+
+``` bash
+kubectl apply -f ./re-production-1.yaml
 ```
-wormhole.cluster.kyma.cx
-nightly.cluster.kyma.cx
+
+## Delete a Remote Environment
+
+The Remote Environment can be removed from Kyma using either Console UI or Kubectl.
```

Comment:
``You can remove a Remote Environment from Kyma using either the Console UI or kubectl.``